www/front_src/src/Resources/Graph/Performance/Graph/index.tsx

```diff
@@ -447,7 +446,10 @@ const GraphContent = ({
     (zoomBoundaries?.end || 0) - (zoomBoundaries?.start || 0),
   );
 
-  const timeValue = getTimeValue(mousePosition?.[0] || 0);
+  const timeTick =
+    containsMetrics && mousePosition
+      ? new Date(getTimeValue(mousePosition[0]).timeTick)
```

Comment:
I'd export the new Date argument into a dedicated variable to increase readibility